### PR TITLE
m2ee-tools 7.2

### PR DESCRIPTION
**Pull request by knorrie**
Everyone invited for review and feedback.

Also see the issues page, many of them get fixes. \o/ 

See CHANGES for a shortlog and NEWS about major changes.

### CODE REVIEW

File: src/m2ee/version.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def __str__(self):
         return version
 
     def __contains__(self, other):
-        if isinstance(other, basestring):
+        if isinstance(other, str):
```
Comment (jtwaleson) [inline src/m2ee/version.py]: str vs. basestring: https://stackoverflow.com/questions/11301138/how-to-check-if-variable-is-string-with-python-2-and-3-compatibility

Including this seems most python2/3 compatible:

    try:
        basestring
    except NameError:
        basestring = str

**Comment by jtwaleson** [comment]
Looks good overall! I added some more changes in #33 .

File: src/m2ee/version.py (hunk 1 above)
Comment (knorrie) [reply src/m2ee/version.py]: I think this only breaks when doing `__future__ import unicode_literals`, but yes, I agree. OTOH, why not simply reverse the check, `if not isinstance(other, MXVersion)`?

Comment (jtwaleson) [reply src/m2ee/version.py]: Hm, yes, the reverse is probably simpler. Added in #33 :)

**Comment by knorrie** [comment]
Ok, great, it's finally going to happen, let's finish the release. Sometimes things don't go exactly like you imagined and priorities change... :| The two times I could work on this in the last months were just before leaving for 2 or 1 week, which is also not really a good idea for this.

I'm preparing the internal release notification / approval, and ask for a Mendix Cloud v3 automated regression testing run. In the meantime I'm collecting some graph data for the added munin graphs and will update the documentation page as very last thing before merging and tagging.

**Pull request merged**